miasm2/jitter/llvmconvert.py

```diff
@@ -307,25 +323,76 @@ def set_vmcpu(self, lookup_table):
     def memory_lookup(self, func, addr, size):
         """Perform a memory lookup at @addr of size @size (in bit)"""
         builder = func.builder
-        fc_name = "MEM_LOOKUP_%02d" % size
-        fc_ptr = self.mod.get_global(fc_name)
-        addr_casted = builder.zext(addr,
-                                   LLVMType.IntType(64))
+        if size <= 64:
+            fc_name = "MEM_LOOKUP_%02d" % size
+            fc_ptr = self.mod.get_global(fc_name)
+            addr_casted = builder.zext(addr, LLVMType.IntType(64))
+            ret = builder.call(
+                fc_ptr, [func.local_vars["jitcpu"],addr_casted]
+            )
+        else:
+            assert size == 128
+
+            fc_name = "MEM_LOOKUP_INT_BN_TO_PTR"
```

Comment:
As the code is quite complicated here, a comment describing the different cast would be welcomed

Reply:
Done